5-network/03-fetch-progress/article.md

```diff
@@ -1,60 +1,60 @@
 
-# Fetch: Download progress
+# Fetch: Postęp pobierania
 
-The `fetch` method allows to track *download* progress.
+Metoda `fetch` pozwala śledzić postęp pobierania (ang. *download*).
 
-Please note: there's currently no way for `fetch` to track *upload* progress. For that purpose, please use [XMLHttpRequest](info:xmlhttprequest), we'll cover it later.
+Należy pamiętać, że `fetch` nie ma możliwości śledzenia postępu wysyłania danych (ang. *upload*). Do tego celu należy użyć [XMLHttpRequest](info:xmlhttprequest). Omówimy to w dalszej części.
 
-To track download progress, we can use `response.body` property. It's  `ReadableStream` -- a special object that provides body chunk-by-chunk, as it comes. Readable streams are described in the [Streams API](https://streams.spec.whatwg.org/#rs-class) specification.
+W celu śledzenia postępu pobierania możemy wykorzystać właściwość `response.body`. Jest to specjalny obiekt `ReadableStream` (pol. *czytelny strumień*), który udostępnia ciało odpowiedzi na bieżąco, kawałek po kawałku (ang. *chunk*). Czytelne strumienie zostały opisane w specyfikacji [Streams API](https://streams.spec.whatwg.org/#rs-class).
```

Comment:
Nie wiem, czy "czytelny" jest odpowiednim słowem na "readable". Chodzi tutaj o strumień dający się odczytywać, a nie że jest ładnie napisany ;-) Chyba że w polskim "czytelny" znaczy też to pierwsze?

Reply:
```suggestion
W celu śledzenia postępu pobierania możemy wykorzystać właściwość `response.body`. Jest to specjalny obiekt `ReadableStream` (pol. *odczytywalny strumień*), który udostępnia ciało odpowiedzi na bieżąco, kawałek po kawałku (ang. *chunk*). Odczytywalne strumienie zostały opisane w specyfikacji [API Strumieni](https://streams.spec.whatwg.org/#rs-class).
```

Reply:
Taaa, gdzieś to znalazłem w necie, ale faktycznie kulawe.